assets/sass/protocol/base/elements/_titles.scss

```diff
@@ -23,6 +23,9 @@ h6 {
 
     @supports (--css: variables) {
         font-family: var(--title-font-family);
+        font-variant-ligatures: var(--title-font-ligatures);
+        font-weight: var(--title-font-weight);
```

Comment:
This is problematic, it overrides the `700` font weight and makes the inherited `500` weight take effect. Needs a default fallback or a Firefox-themed variable value
<img width="1345" alt="Screenshot 2025-03-18 at 6 50 05 PM" src="https://github.com/user-attachments/assets/b9f7b143-9f33-482e-a59c-0df50f84de8f" />

Reply:
default fallback value works as expected

<img width="1041" alt="default-fallback" src="https://github.com/user-attachments/assets/bb13356f-4ad4-461b-a0ed-c190aa890a39" />